util/rbac/rbac.go

```diff
@@ -331,7 +362,11 @@ func (e *Enforcer) SetUserPolicy(policy string) error {
 	e.invalidateCache(func() {
 		e.adapter.userDefinedPolicy = policy
 	})
-	return e.LoadPolicy()
+	err := e.LoadPolicy()
```

Comment:
Is there any reason to not just return the error here?

Reply:
Thanks- this was left over from initial placement of referential validation